<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkServerDialog</class>
 <widget class="QDialog" name="LinkServerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
border-radius:10px;
color:rgb(255, 0, 0);
border:2px solid rgb(0, 255, 0);
}
QLineEdit:hover
{
border:2px solid rgb(255, 0, 0)；
}
</string>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
   <property name="placeholderText">
    <string>IP地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
border-radius:10px;
color:rgb(255, 0, 0);
border:2px solid rgb(0, 255, 0);
}
QLineEdit:hover
{
border:2px solid rgb(255, 0, 0)；
}
</string>
   </property>
   <property name="text">
    <string>12121</string>
   </property>
   <property name="placeholderText">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius:15px;
	background:rgb(87, 255, 227);
}

QPushButton:hover
{
	background:rgb(255, 255, 127)
}</string>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
